neo/test/rawiotest/test_neuralynxrawio.py

```diff
@@ -360,20 +360,18 @@ def test_equality(self):
         ns0.sampFreqUsed = 400
         self.assertNotEqual(ns0, ns1)
 
+class TestNlxHeader(TestNeuralynxRawIO, unittest.TestCase):
+    def test_no_date_time(self):
+        filename = self.get_local_path("neuralynx/NoDateHeader/NoDateHeader.nev")
 
-# I comment this now and will put it back when files will be in gin.g-node
-# class TestNlxHeader(TestNeuralynxRawIO, unittest.TestCase):
-#     def test_no_date_time(self):
-#         filename = self.get_local_path("neuralynx/NoDateHeader/NoDateHeader.nev")
-
-#         with self.assertRaises(IOError):
-#             hdr = NlxHeader(filename)
+        with self.assertRaises(IOError):
+            hdr = NlxHeader(filename)
 
-#         hdr = NlxHeader(filename, props_only=True)
+        hdr = NlxHeader(filename, props_only=True)
 
-#         self.assertEqual(len(hdr), 11)
-#         self.assertEqual(hdr['ApplicationName'], 'Pegasus')
-#         self.assertEqual(hdr['FileType'], 'Event')
+        self.assertEqual(len(hdr), 11)
```

Comment:
For a magic test like this could we get a comment explaining why this should be 11. Is it always 11 or because you made the file you know it is 11? Thanks @PeterNSteinmetz !

Reply:
OK, I added a comment about how there are 9 properties in the file plus the channel_ids and channel_names properties added.